src/Uno.UWP/ApplicationModel/Contacts/Contact.cs

```diff
@@ -5,28 +5,44 @@
 using Uno;
 using Windows.Foundation.Collections;
 using Windows.Storage.Streams;
-
+using Uno.Logging;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+
 namespace Windows.ApplicationModel.Contacts
-{
+{
 	/// <summary>
 	/// Represents a contact.
 	/// </summary>
 	public partial class Contact
 	{
 		[NotImplemented]
-		public string? Id { get; set; }
-
+		public string? Id { get; set; }
+
+		private void LogWarningIfLenExceedsUWPLimit(string value, int lenLimit, string variableName)

```

Comment:
Similar here - please move the method below all other members, as private methods should be last in order